index.html

```diff
@@ -2273,6 +2274,10 @@ <h2>The <dfn>record type</dfn> string</h2>
     The <dfn data-dfn-for="NDEFReader">onreading</dfn> is an {{EventHandler}} which is called to notify
     that new reading is available.
   </p>
+  <p>
+    The <dfn data-dfn-for="NDEFReader">onerror</dfn> is an {{EventHandler}}
+    which is called to notify that an error happened when reading.
```

Comment:
During?

Reply:
You mean "during reading" instead of "when reading"?